video_xblock/video_xblock.py

```diff
@@ -9,19 +9,23 @@
 
 from xblock.core import XBlock
 from xblock.fields import Scope, Boolean, Integer, Float, String
+from xblock.reference.plugins import Filesystem
 from xblock.fragment import Fragment
 from xblock.validation import ValidationMessage
 from xblockutils.studio_editable import StudioEditableXBlockMixin
 
 from django.template import Template, Context
+from django.conf import settings
 
 from backends.base import BaseVideoPlayer, html_parser
-
+from uuid import uuid1
+import base64
```

Comment:
Please reorder imports according to the [guidelines](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#imports-order)